python/adelphi/adelphi/anonymize.py

```diff
@@ -1,12 +1,10 @@
+import re
```

Comment:
Nit: this should be moved below the top-level comment for this package.  At least that's the convention used in the other Adelphi packages.

Reply:
fixed